backend/migrations/20240524154034_incident_data.sql

```diff
@@ -0,0 +1,20 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE incident_data (
+    instance_id VARCHAR(255) NOT NULL,
+    incident_id VARCHAR(255) NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    description VARCHAR(255) NOT NULL,
```

Comment:
```suggestion
    description TEXT NOT NULL,
```

We shouldn't limit the length though

Reply:
Done 👍🏼